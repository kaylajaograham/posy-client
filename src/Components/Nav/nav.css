/* NAV LAYOUT */

.navbar {
    width: 100%;
    list-style: none;
    border-bottom: 1px solid #ccc; 
    border-top: 1px solid #ccc; 
}

.navlinks {
    display: inline-block;
    padding: 8px;
    text-decoration: none;
    font-weight: bold;
    color: peru;
    border-right: 1px solid #ccc; 
}

ul {
    padding: 0;
}
