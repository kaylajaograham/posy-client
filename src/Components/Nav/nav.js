import React from 'react';
import { Link } from 'react-router-dom';

class Nav extends React.Component{
    render() {
        return (
            <div className='navbar'>
              <ul>
                <Link className='navlinks' to='/'>Home</Link>
                <Link className='navlinks' to='/register'>Register</Link>
                <Link className='navlinks' to='/login'>Login</Link>              
              </ul>
            </div>
        );
    }
}

export default Nav;

