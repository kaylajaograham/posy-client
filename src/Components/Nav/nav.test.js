import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Nav from './Nav';

describe(`Nav component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders a Hero by default', () => {
    const wrapper = shallow(<Nav />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the Landing given props', () => {
    const wrapper = shallow(<Nav {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})