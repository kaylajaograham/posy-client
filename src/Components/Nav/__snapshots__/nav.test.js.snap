// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders a Hero by default 1`] = `
<div
  className="navbar"
>
  <ul>
    <Link
      className="navlinks"
      to="/"
    >
      Home
    </Link>
    <Link
      className="navlinks"
      to="/register"
    >
      Register
    </Link>
    <Link
      className="navlinks"
      to="/login"
    >
      Login
    </Link>
  </ul>
</div>
`;

exports[`Nav component renders the Landing given props 1`] = `
<div
  className="navbar"
>
  <ul>
    <Link
      className="navlinks"
      to="/"
    >
      Home
    </Link>
    <Link
      className="navlinks"
      to="/register"
    >
      Register
    </Link>
    <Link
      className="navlinks"
      to="/login"
    >
      Login
    </Link>
  </ul>
</div>
`;
